docs/features/CRAWLER_DIRECTIVES.md

```diff
@@ -0,0 +1,252 @@
+# Crawler Directives
+
+Crawler directives are features that are set up externally that the Open Crawler adheres to.
+These typically require no change to the Open Crawler configuration, and instead changes directly to the webserver or webpage.
+
+These techniques are similar to search engine optimization (SEO) techniques used for other web crawlers and robots.
+For example, you can embed instructions for the web crawler within your HTML content.
+You can also prevent the crawler from following links or indexing any content for certain webpages.
+Use these tools to manage webpage discovery and content extraction.
+
+**Discovery** concerns which web pages and files from crawled domains get indexed:
```

Comment:
```suggestion
**Discovery** concerns which webpages and files from crawled domains get indexed:
```

Reply:
I prefer the two-word form `web page` (as do most dictionaries), but it's reasonable to go with `webpage` in this context. And almost everywhere else, you use `webpage`, so just going for consistency.